apps/system/init/init.c

```diff
@@ -131,6 +134,18 @@ int preapp_start(int argc, char *argv[])
 	tash_register_cmds();
 #endif
 
+#ifdef CONFIG_TASK_MANAGER
+#define TASKMGR_STACK_SIZE 2048
+#define TASKMGR_PRIORITY 200
+	printf("Starting task manager\n");
+
+	pid = task_create("task_manager", TASKMGR_PRIORITY, TASKMGR_STACK_SIZE, task_manager, (FAR char *const *)NULL);
+	if (pid < 0) {
+		printf("Failed to create Task Manager\n");
+		goto error_out;
```

Comment:
below error_out  should be defined for defined(CONFIG_TASK_MANAGER) ...

Reply:
Done. Thanks.